src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -600,12 +602,17 @@ public void calculateContingencySensitivityValues(PropagatedContingency continge
                 if (participatingElementsChanged || rhsChanged) {
                     newFactorStates = calculateStates(j, equationSystem, factorGroups, newParticipatingElements);
                 }
+                // write contingency status
+                resultWriter.writeContingencyStatus(contingency.getIndex(), SensitivityAnalysisResult.Status.CONVERGED); // FIXME
```

Comment:
@Hadrien-Godard here too.

Reply:
Same here. Status FAILED should not appeared in DC sensitivity since we do not track the failure of the LU decomposition like we track the non convergence of the Newton Raphson.